theme/less/_homepage.less

```diff
@@ -0,0 +1,90 @@
+.homepage div
+{
+  grid-template-areas: 
+    "header header"
+    "main main"
+    "footer footer";
+}
+
+.homepage div main article
+{
+  display: grid;
+}
+
+.homepage div main article h2
+{
+  grid-column: 1 e("/") 3;
+}
+
+.box
+{
+  position: relative;
+  background-color: #f2f2f2;
+  border-radius: 8px;
+  border: 1px solid #dcdcdc;
+  padding: 10px 10px 60px 10px;
+  margin: 20px 0 0 0;
+}
+
+.box.first
```

Comment:
You probably can use [first-of-type](https://developer.mozilla.org/en-US/docs/Web/CSS/:first-of-type) or [first-child](https://developer.mozilla.org/en-US/docs/Web/CSS/:first-child) instead of adding specific classname.